svgelements/svgelements.py

```diff
@@ -5402,6 +5441,8 @@ def __init__(self, *args, **kwargs):
             elif isinstance(s, PathSegment):
                 self._segments.append(s)
         if SVG_ATTR_DATA in self.values:
+            # Not sure what the purpose of pathd_loaded is.
+            # It is only set and checked here and you cannot have "d" attribute more than once anyway
```

Comment:
There's a very rare edge condition with a `d` namespace. These show up in the WC3 tests. This tiny code was intended to avoid the weird error it introduced.

Reply:
This is why I didn't remove it - I assumed that there was something non-obvious.